### DISCUSSION TITLE
Connection pool for postgresql

### DISCUSSION THREAD

**Discussion by jcm-insens** [ibis-project/ibis#8663] [Q&A] [answered]
Hi, 

I need to use ibis for a high performance application.

Is it possible to use a connection pool (other than SqlAlchemy's StaticPool) with ibis?

Thank for your help!

**Comment by cpcloud** [comment, marked as answer]
We don't use SQLAlchemy anymore and we are now connecting directly to backend resources using driver APIs. An instance of an Ibis backend has a single connection that it owns.

Currently there's no built-in support for connection pooling.
